cpp/cmake_modules/FindClangTools.cmake

```diff
@@ -28,12 +28,21 @@
 #  CLANG_TIDY_BIN, The  path to the clang tidy binary
 #  CLANG_TIDY_FOUND, Whether clang tidy was found
 #  CLANG_FORMAT_BIN, The path to the clang format binary
-#  CLANG_TIDY_FOUND, Whether clang format was found
+#  CLANG_FORMAT_FOUND, Whether clang format was found
 
 if (DEFINED ENV{HOMEBREW_PREFIX})
   set(HOMEBREW_PREFIX "$ENV{HOMEBREW_PREFIX}")
 else()
-  set(HOMEBREW_PREFIX "/usr/local")
+  find_program(BREW_BIN brew)
+  if ((NOT ("${BREW_BIN}" STREQUAL "BREW_BIN-NOTFOUND")) AND APPLE)
+    execute_process(
+      COMMAND ${BREW_BIN} --prefix
+      OUTPUT_VARIABLE HOMEBREW_PREFIX
+      OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+  else()
+    set(HOMEBREW_PREFIX "/usr/local")
+  endif()
```

Comment:
I don't think clang-format-6 is available in Homebrew unfortunately

Reply:
I confirmed that `llvm@6` has clang-format-6.0.1 👍

Reply:
Ah that's great. Let's definitely not encourage anyone to use the clang-format recipe then =) https://github.com/Homebrew/homebrew-core/blob/master/Formula/clang-format.rb